src/core/transaction.js

```diff
@@ -1413,6 +1461,24 @@ Transaction.prototype.summary = function() {
   return txSummary;
 };
 
-exports.Transaction = Transaction;
+Transaction.prototype.addSigner = function(signer) {
+  if (_.isUndefined(this.signers)) {
+    this.signers = [];
+  }
+  this.signers.push({Signer: signer});
```

Comment:
Haven't confirmed that it matters. rippled responds with a tesSUCCESS & sorted signers